### PR TITLE
Add CRUDL support for environment tags

### PR DESCRIPTION
**Pull request by cleverguy25**
Most of this is generated from changes to swagger. The main files that changed were:

- sdk/swagger.yaml
- sdk/go/api_esc_extensions.go
- sdk/go/api_esc_test.go
- sdk/python/pulumi_esc_sdk/esc_client.py
- sdk/python/test/test_esc_api.py
- sdk/typescript/esc/index.ts
- sdk/typescript/test/client.spec.ts

### CODE REVIEW

File: sdk/swagger.yaml [hunk 1]
```diff
@@ -177,6 +177,74 @@ components:
         schema: {}
         executionContext:
           $ref: "#/components/schemas/EvaluatedExecutionContext"
+    EnvironmentTag:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
```
Comment (dschaller) [inline sdk/swagger.yaml:185]: we dropped this from the api response so I think we can ignore it here too

File: sdk/swagger.yaml [hunk 2]
```diff
@@ -177,6 +177,74 @@ components:
         schema: {}
         executionContext:
           $ref: "#/components/schemas/EvaluatedExecutionContext"
+    EnvironmentTag:
+      type: object
+      properties:
+        id:
```
Comment (dschaller) [inline sdk/swagger.yaml:183]: we dropped this from the api response so I think we can ignore it here too

File: sdk/swagger.yaml [hunk 3]
```diff
@@ -427,6 +495,30 @@ components:
...
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ListEnvironmentTags"
+    CreateEnvironmentTag:
+      description: Create environment tag
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/CreateEnvironmentTag"
+    UpdateEnvrionmentTag:
```
Comment (dschaller) [inline sdk/swagger.yaml:510]:
```suggestion
    UpdateEnvironmentTag:

```

File: sdk/swagger.yaml [hunk 4]
```diff
@@ -427,6 +495,30 @@ components:
...
+      description: Update environment tag
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/UpdateEnvironmentTag"
+    EnvironmentTag:
+      description: Create environment tag
```
Comment (dschaller) [inline sdk/swagger.yaml:517]:
```suggestion
      description: Get an environment tag

```

**Review by dschaller — APPROVED** [review]

File: sdk/swagger.yaml (hunk 4 above)
Comment (cleverguy25) [reply sdk/swagger.yaml:517]: This is the response from create environment tag

File: sdk/swagger.yaml (hunk 2 above)
Comment (cleverguy25) [reply sdk/swagger.yaml]: fixed

File: sdk/swagger.yaml (hunk 3 above)
Comment (cleverguy25) [reply sdk/swagger.yaml]: fixed

**Pull request merged**